relay-server/src/utils/unreal.rs

```diff
@@ -481,4 +504,17 @@ mod tests {
 
         insta::assert_snapshot!(Annotated::new(event).to_json_pretty().unwrap());
     }
+
+    #[test]
+    fn test_merge_unreal_context_login_id() {
+        let context = get_context_with_login_id();
+        let mut event = Event::default();
+
+        merge_unreal_context(&mut event, context);
+
+        assert_eq!(
+            event.user.0.unwrap().id.0.unwrap(),
+            LenientString("SOME_ID".to_string())
```

Comment:
```suggestion
            LenientString("SOME_ID".to_owned())
```

`to_owned` expresses the intention better, a `&str` is conceptually already a string